### ISSUE TITLE
c_templates dependency

### ISSUE THREAD

**Issue by Owlz** [shellphish/rex#1]
Hey,

Trying to import rex, i get a dependency issue:

```python
In [1]: import rex
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-e841dd4a92c3> in <module>()
----> 1 import rex

/home/user/.virtualenvs/angr/local/lib/python2.7/site-packages/rex/__init__.py in <module>()
      1 from rex.vulnerability import Vulnerability
----> 2 from rex.crash import Crash, QuickCrash, NonCrashingInput
      3 from rex.exploit import Exploit, CannotExploit, CannotExplore

/home/user/.virtualenvs/angr/local/lib/python2.7/site-packages/rex/crash.py in <module>()
     11 import operator
     12 from .trace_additions import ChallRespInfo, ZenPlugin
---> 13 from rex.exploit import CannotExploit, CannotExplore, ExploitFactory, CGCExploitFactory
     14 from rex.vulnerability import Vulnerability
     15 from simuvex import SimMemoryError, s_options as so

/home/user/.virtualenvs/angr/local/lib/python2.7/site-packages/rex/exploit/__init__.py in <module>()
      3 from .shellcode_factory import ShellcodeFactory
      4 from .exploit import Exploit, ExploitException
----> 5 from .exploit_factory import ExploitFactory
      6 from .cgc_exploit_factory import CGCExploitFactory
      7 from .techniques import Techniques

/home/user/.virtualenvs/angr/local/lib/python2.7/site-packages/rex/exploit/exploit_factory.py in <module>()
     58                 l.debug("technique failed: %s", e.message)
     59 
---> 60 from .techniques import Techniques

/home/user/.virtualenvs/angr/local/lib/python2.7/site-packages/rex/exploit/techniques/__init__.py in <module>()
      8 from ..technique import Technique
      9 
---> 10 from .rop_set_register import RopSetRegister
     11 from .shellcode_set_register import ShellcodeSetRegister
     12 from .circumstantial_set_register import CircumstantialSetRegister

/home/user/.virtualenvs/angr/local/lib/python2.7/site-packages/rex/exploit/techniques/rop_set_register.py in <module>()
      4 from rex import Vulnerability
      5 from rex.exploit import CannotExploit
----> 6 from ..cgc import CGCType1RopExploit
      7 from ..technique import Technique
      8 

/home/user/.virtualenvs/angr/local/lib/python2.7/site-packages/rex/exploit/cgc/__init__.py in <module>()
----> 1 from .cgc_type1_exploit import CGCType1Exploit
      2 from .cgc_type2_exploit import CGCType2Exploit
      3 from .cgc_exploit import CGCExploit
      4 from .type1 import CGCType1RopExploit, CGCType1ShellcodeExploit, CGCType1CircumstantialExploit
      5 from .type2 import CGCType2RopExploit, CGCType2ShellcodeExploit

/home/user/.virtualenvs/angr/local/lib/python2.7/site-packages/rex/exploit/cgc/cgc_type1_exploit.py in <module>()
      8 import tempfile
      9 from .cgc_exploit import CGCExploit
---> 10 from .c_templates import c_template_type1
     11 
     12 import logging

ImportError: No module named c_templates
```

I have no idea where c_templates is.

**Comment by Owlz** [comment]
Ok. So looks like if i run it from the rex directory it works. The c_templates module apparently isn't being installed w/ the rest of the package. Guessing this is an update to the setup.py file, but I'm not well versed in pypy setup stuff.

**Comment by Owlz** [comment]
type1 and type2 directories will need to be installed as well.

**Comment by zardus** [comment]
Should be fixed in e7ae2322b299face9a1209a4ce3f097a5f9522e1!
